supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -21,6 +21,7 @@ object Run {
 
   def makeRunPipelineRequest(jobDescriptor: BackendJobDescriptor,
                              runtimeAttributes: JesRuntimeAttributes,
+                             dockerImage: String,
```

Comment:
But we do want to allow tags here if hash lookup failed and that's all we've got.